src/main/fulcro/client/impl/protocols.cljc

```diff
@@ -13,6 +13,7 @@
   (get-id [this] "Get the unique ID of this reconciler. Used to resolve multiple running apps on a page. Defaults to the root class and a UUID.")
   (get-history [this] "Returns the current history atom")
   (get-network-activity [this] "Returns an atom containing network activity info.")
+  (get-load-marker-default [this])
```

Comment:
Please do not try to make these protocols bigger.   I plan on removing them someday because they are absolutely senseless in the context of Fulcro and came from Om Next.  The reconciler is a defrecord, which means that any "field" you put in it is publicly accessible just like a map.  Please just use the keys. This is all internal implementation anyhow.